flow/table.go

```diff
@@ -582,14 +584,18 @@ func (ft *Table) Run() {
 			if ft.ipDefragger != nil {
 				ft.ipDefragger.FlushOlderThan(t)
 			}
-		case <-statusTicker.C:
-			ft.status.CaptureID = ft.uuids.CaptureID
-			ft.status.FlowCount = int64(ft.table.Len())
-
-			ft.sender.SendStatus(ft.status)
-
-			logging.GetLogger().Warningf("Flow table status: %+v", ft.status)
-			ft.status = Status{}
+		case s := <-ft.statsChan:
+			ft.stats.KernelFlowDropped += s.KernelFlowDropped
+			ft.stats.PacketsReceived += s.PacketsReceived
+			ft.stats.PacketsDropped += s.PacketsDropped
```

Comment:
This looks like a candidate for an extracted function:

    func (s *Stats) merge(other *Stats)

Can we add methods on `Stats` even when it's generated from a proto file?

Reply:
Only used here, no real point to extract for now and the function won't be generic as only few metrics are "merged"